### ISSUE TITLE
Add "Change Presets" to the menu

### ISSUE THREAD

**Issue by barefootstache** [Helium314/SCEE#433]
**Issue**

Currently changing presets is either hidden under the *Settings* or under the *Quick Settings* UI:

- under the Settings: the problem is that it takes a couple of clicks to navigate to it and then change it and more times than not I go to the profile in hope to change the presets there.
- under the Quick Settings: this technically is one solution, though it bloats the screen

**Proposed Solutions**

(Any of these solutions, if not a combination are fine.)

*Proposed Solution 1*

Create another list item under the "Enter Team Mode" item as seen in the image:

![Screenshot_2023-06-18_15-01-27](https://github.com/Helium314/SCEE/assets/104735442/43029678-73d2-49bc-aff2-c7752edeb447)

*Proposed Solution 2*

Add another item to the grid view in this modal:

![Screenshot_2023-06-18_15-05-46](https://github.com/Helium314/SCEE/assets/104735442/a76a8ad6-64be-4b71-820e-b783a0ee4632)

*Proposed Solution 3*

I feel like the "Enter Team Mode" is rarely used and replacing it completely with "Changing Presets" feature.

*Optional proposed addition*

Make it possible that the user chooses how they would like to have menu appear. This feature would most likely take the most time to implement and is not a burning want.

**Comment by Helium314** [comment]
This should definitely be optional, as SCEE should be as close to SC as possible by default (with exception of the star count...), and changing the default menu is annoying as it will affect all button positions on screen.

Not sure when I'll come around to implement this, as it's a rather low priority for me, but this shouldn't be too much work.

**Comment by Helium314** [comment]
Done in https://github.com/Helium314/SCEE/commit/670dc94016e279c4a650a1c4b842bf3c345efbc1
Didn't test it though, you can try it once the build is done: https://github.com/Helium314/SCEE/actions/runs/5422038673
